habitat-baselines/habitat_baselines/config/experiments_hab3/socialnav_human_robot_oracle.yaml

```diff
@@ -0,0 +1,194 @@
+  - /habitat/task/actions@habitat.task.actions.agent_0_arm_action: arm_action
+  - /habitat/task/actions@actions.agent_0_oracle_nav_action: oracle_nav_action
+  - /habitat/task/actions@actions.agent_1_oracle_nav_action: oracle_nav_action
+  - override /habitat/task/rearrange: multi_agent_tidy_house_base
+  # For Spot observation space
+  - override /habitat/simulator/agents@habitat.simulator.agents.agent_0: rgbd_head_rgbd_arm_agent
+  - _self_
+
+habitat:
+  task:
+    reward_measure: social_nav_reward
+    slack_reward: 0.
+    success_reward: 0.
+    task_spec: multi_agent_social_nav_solved
+    actions:
+      agent_0_arm_action:
+        grip_controller: MagicGraspAction
+      agent_0_oracle_nav_action:
+        agent_index : 0
+        spawn_max_dist_to_obj: -1.0
+      agent_1_oracle_nav_action:
+        motion_control: human_joints
+        spawn_max_dist_to_obj: 1.7
+    measurements:
+      cooperate_subgoal_reward:
+        end_on_collide: False
+
+  environment:
+    max_episode_steps: 700
+
+  gym:
+    obs_keys:
+      - agent_0_head_depth
+      - agent_0_relative_resting_position
+      - agent_0_obj_start_sensor
+      - agent_0_obj_goal_sensor
+      - agent_0_obj_start_gps_compass
+      - agent_0_obj_goal_gps_compass
+      - agent_0_joint
+      - agent_0_is_holding
+      - agent_0_ee_pos
+      - agent_0_localization_sensor
+      - agent_1_head_depth
+      - agent_1_relative_resting_position
+      - agent_1_obj_start_sensor
+      - agent_1_obj_goal_sensor
+      - agent_1_obj_start_gps_compass
+      - agent_1_obj_goal_gps_compass
+      - agent_1_is_holding
+      - agent_1_ee_pos
+      - agent_1_localization_sensor
+      - agent_1_humanoid_joint_sensor
+  simulator:
+    agents:
+      agent_0:
+        radius: 0.3
+        articulated_agent_urdf: data/robots/hab_spot_arm/urdf/hab_spot_arm.urdf
+        articulated_agent_type: "SpotRobot"
+        sim_sensors:
+          arm_rgb_sensor:
+            height: 480
+            width: 640
+            hfov: 47
+          arm_depth_sensor:
+            height: 224
+            width: 171
+            hfov: 55
+      agent_1:
+        articulated_agent_urdf: 'data/humanoids/humanoid_data/armatures/human_armature.urdf'
+        articulated_agent_type: KinematicHumanoid
+        motion_data_path: "data/humanoids/humanoid_data/walking_motion_processed.pkl"
+
+    kinematic_mode: True
+    ac_freq_ratio: 1
+    step_physics: False
+    habitat_sim_v0:
+      allow_sliding: True
+
+habitat_baselines:
+  verbose: False
+  trainer_name: "ddppo"
+  updater_name: "PPO"
+  distrib_updater_name: "DDPPO"
+  torch_gpu_id: 0
+  video_fps: 30
+  eval_ckpt_path_dir: "data/checkpoints"
+  num_environments: 1
+  num_updates: -1
+  total_num_steps: 5.0e7
+  log_interval: 10
+  num_checkpoints: 10
+  force_torch_single_threaded: True
+  eval_keys_to_include_in_name: ['composite_success']
+  load_resume_state_config: False
+  rollout_storage_name: "HrlRolloutStorage"
```

Comment:
Use the regular RolloutStorage if you aren't using any hierarchical policy training.